errors/errors.go

```diff
@@ -10,70 +10,69 @@ import (
 // with the registry API
 const errGroup = "notary.api.v1"
 
+// These errors should be returned from contextHandlers only. They are
+// serialized and returned to a user as part of the generic error handling
+// done by the rootHandler
 var (
-	// ErrNoStorage lint comment
```

Comment:
@dmcgowan worked out how to get rid of all these comments. Apparently lint accepts a single comment before the var/const block :-)